security/pam-reattach/Portfile

```diff
@@ -0,0 +1,36 @@
+# -*- coding: utf-8; mode: tcl; tab-width: 4; indent-tabs-mode: nil; c-basic-offset: 4 -*- vim:fenc=utf-8:ft=tcl:et:sw=4:ts=4:sts=4
+
+PortSystem          1.0
+PortGroup           cmake 1.1
+PortGroup           github 1.0
+
+name                pam-reattach
+github.setup        fabianishere pam_reattach 1.2 v
+revision            0
+
+categories          security
+license             MIT
+maintainers         {@kakuhen} openmaintainer
+platforms           darwin
+supported_archs     x86_64 arm64
```

Comment:
any reason this wouldn't work on `i386`, if not just remove the line and stay with the default

Reply:
Makes use of XPC, which is available only on OS X 10.7 and later. I removed the architecture line and added a version check to the Portfile.